frontend/catalyst/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.10.0"
+__version__ = "0.11.0-dev"
```

Comment:
```suggestion
__version__ = "0.11.0-dev0"
```

See https://github.com/PennyLaneAI/catalyst/pull/1296

Reply:
Good catch, thanks! I'll update the guidance docs to make sure the `0` is added for future releases.